static/app/views/usageStats/types.tsx

```diff
@@ -1,28 +0,0 @@
-import {SeriesApi} from 'app/types';
```

Comment:
Not actually deleting this file.

It was moved to `static/app/views/organizationStats/types.tsx` (8 files above this) but git didn't resolve the file movement correctly.